cmake/Modules/FindTriSYCL.cmake

```diff
@@ -0,0 +1,93 @@
+#.rst:
+# FindTriSYCL
+#---------------
+#
+#   TODO : add Copyright
+
+#########################
+#  FindTriSYCL.cmake  
+#########################
+#
+#  Tools for finding and building with triSYCL.
+#
+#  User must define:
+#	TRISYCL_INCLUDE_DIR pointing to triSYCL
+#		available : https://github.com/keryell/triSYCL
+#	BOOST_COMPUTE_INCLUDE_DIR pointing to Compute
+#		available : https://github.com/boostorg/compute
+#
+
+# Require CMake version 3.2.2 or higher
+cmake_minimum_required(VERSION 3.2.2)
+
+# Check that a supported host compiler can be found
+if(CMAKE_COMPILER_IS_GNUCXX)
+    # Require at least gcc 4.8
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.8)
+      message(FATAL_ERROR
+        "host compiler - Not found! (gcc version must be at least 4.8)")
+    # Require the GCC dual ABI to be disabled for 5.1 or higher
+    elseif (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 5.1)
+      set(COMPUTECPP_DISABLE_GCC_DUAL_ABI "True")
```

Comment:
This is probably not needed on triSYCL since you are not distributing binaries built with the older version of the GCC ABI